tests/server.test.js

```diff
@@ -83,53 +97,86 @@ describe('Artist Routes', () => {
     expect(artist.status).toBe(200);
     expect(artist.body.genres).toEqual(['R&B', 'Soul', 'Alternate R&B']);
   });
+
+  test('Deletes a single artist', async () => {
+    await request.delete('/artist/1');
+    const response = await request.get('/artist');
+    expect(response.body[0].name).toEqual('Giveon');
+    expect(response.body[0].single).toEqual('For Tonight');
```

Comment:
Change the `Griven` to `JVKE` and `For Tonight` to `Golden Hour` because Griven dose does not exist anymore, so JVKE is the new [0]